test_conformance/c11_atomics/main.cpp

```diff
@@ -36,6 +36,7 @@ bool gFloatAtomicsSupported = false;
 cl_device_fp_atomic_capabilities_ext gHalfAtomicCaps = 0;
 cl_device_fp_atomic_capabilities_ext gDoubleAtomicCaps = 0;
 cl_device_fp_atomic_capabilities_ext gFloatAtomicCaps = 0;
+cl_device_fp_config gHalfCaps = 0;
```

Comment:
Nitpick: These aren't "half caps", rather they're the "half FP config".


```suggestion
cl_device_fp_config gHalfFPConfig = 0;
```